.jx/updatebot.yaml

```
@@ -10,3 +10,11 @@ spec:
             pattern: "<alfresco-process-sdk.version>(.*)</alfresco-process-sdk.version>"
             files:
               - pom.xml
+    - urls:
```

I think you can simply add the process-release URL in the first block without creating a new one
```suggestion
    - urls:
            - https://github.com/Alfresco/alfresco-process-acceptance-tests
            - https://github.com/Alfresco/alfresco-process-releases
```

---

so know this will work right? 
I haven't seen it before, but I trust you 😉

---

you can check it here https://github.com/Alfresco/alfresco-modeling-service/blob/develop/.jx/updatebot.yaml 😉